package.json

```diff
@@ -6,6 +6,7 @@
         "auth0-js": "^9.6.0",
         "fbemitter": "^2.1.1",
         "firebase": "^5.2.0",
+        "lodash": "^4.17.10",
```

Comment:
Cały lodash? Toż to będzie ważyć .. dużo.
Użyjmy tree shakingu, taka dobra rada :)

Reply:
https://github.com/warsawjs/topics-manager/pull/27/commits/3e1300ff4253d376e44e87e5f39528caae8a761f & https://github.com/warsawjs/topics-manager/pull/27/commits/28db10d215f21a6294e750d02c8b39668659e604